### PR TITLE
[ACS-5166] Changes to Digital workspace new UI for better experience

### PR DESCRIPTION
**Pull request by arohilaGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACS-5166

**What is the new behaviour?**

Design changes to new UI layout

**Does this PR introduce a breaking change?** (check one with "x")

> - [x] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

https://github.com/Alfresco/alfresco-ng2-components/pull/8640, must be merged before merging this PR.

**Other information**:

### CODE REVIEW

File: projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html [hunk 1]
```diff
@@ -4,10 +4,9 @@
       tabindex=0
       (click)="toggleNavBar.emit()"
       (keypress)="toggleNavBar.emit()">
-      <img
```
Comment (DenysVuika) [inline projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:7]: you are killing the existing feature to display custom logo that customers can change

**Review by DenysVuika — CHANGES_REQUESTED** [review]

**Comment by DenysVuika** [comment]
@arohilaGL please provide jira link in the PR description, and also screenshots of the end results. Based on my tests, the icons are broken in this PR

File: projects/aca-content/assets/app.extensions.json [hunk 2]
```diff
@@ -295,7 +296,7 @@
             "order": 200,
             "title": "APP.NEW_MENU.MENU_ITEMS.CREATE_LIBRARY",
             "description": "APP.NEW_MENU.TOOLTIPS.CREATE_LIBRARY",
-            "icon": "create_new_folder",
+            "icon": "adf:create_library",
```
Comment (DenysVuika) [inline projects/aca-content/assets/app.extensions.json:299]: where this icon is coming from? I don't see this registered anywhere

File: projects/aca-content/assets/app.extensions.json [hunk 3]
```diff
@@ -306,7 +307,7 @@
           {
             "id": "app.create.fileFromTemplate",
             "order": 300,
-            "icon": "description",
+            "icon": "adf:file_from_template",
```
Comment (DenysVuika) [inline projects/aca-content/assets/app.extensions.json:310]: missing icon

File: projects/aca-content/assets/app.extensions.json [hunk 4]
```diff
@@ -320,7 +321,7 @@
           {
             "id": "app.create.folderFromTemplate",
             "order": 400,
-            "icon": "create_new_folder",
+            "icon": "adf:create_new_folder",
```
Comment (DenysVuika) [inline projects/aca-content/assets/app.extensions.json:324]: why adf prefix here? is that a custom icon?

**Comment by DenysVuika** [comment]
@arohilaGL if this PR depends on other PRs, you should state this in the description. This PR relies on ADF changes

File: projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html (hunk 1 above)
Comment (arohilaGL) [reply projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:7]: image
It is a change suggested in https://alfresco.atlassian.net/browse/ACS-5166. To replace the logo with collapse icon.

Comment (DenysVuika) [reply projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:7]: if this is approved, you need to cleanup the code that used to do the logo$ and everything related to that logo and not leave unused code in the repo

File: projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html [hunk 5]
```diff
@@ -4,10 +4,9 @@
...
-        src="{{ logo$ | async }}"
+        <adf-icon value="adf:side_nav_collapse"
         title="{{'APP.TOOLTIPS.COLLAPSE_NAVIGATION' | translate}}"
-        alt="{{ 'CORE.HEADER.LOGO_ARIA' | translate }}" />
```
Comment (DenysVuika) [inline projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:10]: if you are removing an attribute with i18n key, you should check if it is no longer used anywhere, and remove the key from the `en.json` file too

Comment (arohilaGL) [reply projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:10]: It is being used elsewhere

File: projects/aca-content/assets/app.extensions.json (hunk 4 above)
Comment (arohilaGL) [reply projects/aca-content/assets/app.extensions.json]: this has been changed to use pre-existing icons

File: projects/aca-content/assets/app.extensions.json (hunk 3 above)
Comment (arohilaGL) [reply projects/aca-content/assets/app.extensions.json]: this has been changed to use pre-existing icons

File: projects/aca-content/assets/app.extensions.json (hunk 2 above)
Comment (arohilaGL) [reply projects/aca-content/assets/app.extensions.json]: this has been changed to use pre-existing icons

File: projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html (hunk 1 above)
Comment (arohilaGL) [reply projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html:7]: hi @DenysVuika , there was a miscommunication, which caused this change. We have made the right changes now. you can please review the PR

**Review by DenysVuika — APPROVED** [review]

**Comment by DenysVuika** [comment]
@arohilaGL create library seems to be missing the icon, also the menu is trancated
image

File: app/src/assets/fonts/material-icons/MaterialSymbolsOutlined.ttf
Comment (DenysVuika) [inline app/src/assets/fonts/material-icons/MaterialSymbolsOutlined.ttf:1]: we already have material icons font, you are adding a second font there that is "Material Symbols", not "Material Icons"

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: app/project.json [hunk 6]
```diff
@@ -95,6 +95,7 @@
         ],
         "styles": [
           "app/src/assets/fonts/material-icons/material-icons.css",
+          "app/src/assets/fonts/material-icons/material-icons-outlined.css",
```
Comment (DenysVuika) [inline app/project.json:98]: you are loading additional font on top of existing font in line 97

File: projects/aca-shared/src/lib/components/tool-bar/toolbar-button/toolbar-button.component.html [hunk 7]
```diff
@@ -16,6 +16,7 @@
     <button
       [id]="actionRef.id"
       [color]="data?.color || color"
+      [ngClass]="{'secondary-button-color': !data?.color}"
```
Comment (DenysVuika) [inline projects/aca-shared/src/lib/components/tool-bar/toolbar-button/toolbar-button.component.html:19]: you set the "primary" color in the app config, but this is not reflected as you just force a static background

File: projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss [hunk 8]
```diff
@@ -24,33 +24,25 @@
...
-          align-items: center;
-          border-radius: 6px;
-          height: 32px;
-          line-height: 32px;
-
-          .mat-button-wrapper {
-            display: flex;
-          }
+        .secondary-button-color {
+          background: var(--theme-app-toolbar-button-background-color);
```
Comment (DenysVuika) [inline projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss:44]: how does the "primary" color in the app config matter here?

**Comment by DenysVuika** [comment]
reworking the PR (https://github.com/Alfresco/alfresco-content-app/pull/3287) so closing this one

**Pull request closed without merging**